subsys/net/ip/net_context.c

```diff
@@ -1265,7 +1265,8 @@ static int context_sendto(struct net_context *context,
 			  socklen_t addrlen,
 			  net_context_send_cb_t cb,
 			  s32_t timeout,
-			  void *user_data)
+			  void *user_data,
+			  bool sendto)
```

Comment:
This should not be needed, as send() is strictly equivalent to sendto(dst_addr=NULL). So, the right solution would be to drop net_offload_send().

Reply:
There is definitely things to improve on net_offload and its integration within net_context but it's too late for now. So let's keep such ideas for 1.15